apps/mobile/src/app/(home)/_layout.tsx

```diff
@@ -126,28 +126,28 @@ export default function StackLayout() {
       />
       <Stack.Screen
         name="wallet-settings/index"
-        options={{ header: () => NavigationWalletSettings() }}
+        options={{ header: () => WalletNavigationSettings() }}
       />
       <Stack.Screen
-        name="wallet-settings/hidden-accounts"
-        options={{ header: () => NavigationWalletSettings() }}
+        name="settings/wallet/hidden-accounts"
```

Comment:
Can these names use the values in `apps/mobile/src/routes.ts`? to avoid repeating `settings/wallet/`, `settings/wallet/configure`